coalib/results/SourceRange.py

```diff
@@ -120,6 +120,47 @@ def expand(self, file_contents):
                                        tr.end.line,
                                        tr.end.column)
 
+    @enforce_signature
+    def affected_source(self, file_dict: (dict, None)):
```

Comment:
Because there are some test cases whose `file_dict`(`args[0]`) is None. maybe I should handle the `None` case inside `ConsoleInteraction.py`. But I thought handling `None` inside `affected_source` is more meaningful. What do you say ?